cg/models/demultiplex/run_parameters.py

```diff
@@ -19,6 +29,7 @@ def __init__(self, run_parameters_path: Path):
         self.path: Path = run_parameters_path
         self.tree: ElementTree = read_xml(file_path=run_parameters_path)
         self.validate_instrument()
+        self._index_settings: IndexSettings = self._get_index_settings()
```

Comment:
I think this can just be 
```suggestion
        self.index_settings: IndexSettings = self._get_index_settings()
```
and then skip the property. Otherwise this is exactly what I meant 👍

Reply:
fixed